package/yast2.spec

```diff
@@ -177,6 +177,14 @@ mkdir -p %{buildroot}%{_sysconfdir}/YaST2
 %post
 %{fillup_only -n yast2}
 
+if [ -f "/etc/sysctl.d/30-yast.conf" ]; then
+    if [ -f "/etc/sysctl.d/70-yast.conf" ]; then
+        rm /etc/sysctl.d/30-yast.conf
```

Comment:
From the PR's description 

> If the 70-yast.conf already exists, the 30-yast.conf will be simply removed **(this case should not happen)**.

If we are sure that it will not happen, I'd simplify this.

Anyway, it looks good to me as it is.